consensus/beacon/consensus.go

```diff
@@ -105,12 +105,27 @@ func errOut(n int, err error) chan error {
 	return errs
```

Comment:
Can you add a comment on line 61 of this file that eth1 can be optimism pre-bedrock (or something to that effect)